src/python/ensembl/io/genomio/genbank/extract_data.py

```diff
@@ -347,10 +351,10 @@ def _uniquify_id(self, gene_id, all_ids):
         new_id = gene_id
         num = 1
         while new_id in all_ids:
-            print(f"{new_id} exists, update")
+            logging.info(f"{new_id} already exists, append a number to the new one to make it unique")
```

Comment:
```suggestion
            logging.info(f"{new_id} already exists: appending a number to the new one to make it unique")
```
And I would move this message before the loop, as I believe notifying once and then later what number it has been should be enough.